test/cli/kit/system_test.rb

```diff
@@ -52,11 +52,26 @@ def test_system_finds_ruby_script_with_path_modifications
         CLI::Kit::System.fake("ruby", "-e", "puts 'system ruby'", allow: true)
 
         with_script_in_tmpdir("ruby") do |tmpdir|
-          with_env("PATH" => "#{tmpdir}:#{ENV['PATH']}") do
+          path = tmpdir
+          with_env("PATH" => "#{path}#{File::PATH_SEPARATOR}#{ENV['PATH']}") do
```

Comment:
I can't see the point of `path`. Is there one?
```suggestion
          with_env("PATH" => "#{tmpdir}#{File::PATH_SEPARATOR}#{ENV['PATH']}") do
```

Reply:
It's not necessary at all. That line did something and I removed it, but forgot to remove the variable.